source/agora/node/Runner.d

```diff
@@ -215,18 +226,20 @@ private void setVibeLogLevel (ILogger.Level level) @safe
 
 *******************************************************************************/
 
-private TLSContext getTLSContext ()
+private TLSContext getTLSContext (Logger log)
 {
     const cert_file = "agora-cert.pem";
     const cert_search_paths = [
         "./",
+        "/etc/ssl/",
```

Comment:
Own commit with explanation as to why this addition is necessary.

Reply:
I removed as not really needed as current folder can be used.